### PR TITLE
tito build --test --rpm -i Traceback on Fedora 31

### PR DESCRIPTION
**Pull request by schlupov**
Fixes: #341

### CODE REVIEW

File: src/tito/builder/main.py [hunk 1]
```diff
@@ -1340,7 +1340,12 @@ def is_installed(self, package, version):
         if not q:
             return False
 
-        iv = "%s-%s" % (q.version.decode("utf-8"), q.release.decode("utf-8"))
+        if isinstance(q.version, bytes):
+            iv = "%s-%s" % (q.version.decode("utf-8"), q.release.decode("utf-8"))
```
Comment (FrostyX) [inline src/tito/builder/main.py]: It looks like this is the only line, that conditionally changes. The `iv_short` definition and `return` lines are the same for both cases. IMHO we can simplify this.

Comment (schlupov) [reply src/tito/builder/main.py]: Ok, I changed that condition.

Comment (FrostyX) [reply src/tito/builder/main.py]: This looks much better, thank you.
Also, there seems to be a function `compat.ensure_text`, that might be useful in this situation.

**Comment by dgoodwin** [comment]
Thanks @schlupov

**Pull request merged**